### ISSUE TITLE
Missing or wrong aria-labels

### ISSUE THREAD

**Issue by SaiShan** [SAP/fundamental-ngx#914]
#### Is this a bug, enhancement, or feature request?
Bug (accessibility)

#### Briefly describe your proposal.
- [ ] If a user tabs to the objects, the user must know what element he has selected and how many other items are in the current group. (ie. "Option 1, item 1 of 4")
       - Combobox input
       - Dropdown
       - Menu
       - Side navigation

- [ ] If the user tabs into the ‘Right Aligned Text Addon’ element in the Input group component, he only can hear ‘Field placeholder text edit’ instead of ‘Right Aligned Text Addon edit’.

- [ ] The Multi Input reads as ‘image label as popup select fruit edit type in text’ Instead it should read as ‘Select fruit edit type in text’.

**Comment by MattL75** [comment]
We don't have a localization editor, nor a tree (sorta)
